core/scripts/chaincli/.env.debugging.example

```diff
@@ -8,8 +8,8 @@ KEEPER_REGISTRY_ADDRESS=<Registry Address>
 #TENDERLY_ACCOUNT_NAME=<Tenderly Account Name>
 #TENDERLY_PROJECT_NAME=<Tenderly Project Name>
 
-# [Optional] add mercury info only if your upkeep uses mercury
-#MERCURY_ID=<Mercury ID>
```

Comment:
will we do these renaming in core nodes too?
if so, lots of node configs need to change too.

Reply:
you are right, there are a ton of mercury in our codebase.

The answer is no, at least at this point. in this PR, we are just changing the external facing names. For example, we expect users who use our debugging script will need to provide this Mercury_ID, hence renaming it.